TestVectors/Ed25519Signature2020/signedEdSig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "@context": [
+    "https://www.w3.org/ns/credentials/v2",
+    "https://www.w3.org/ns/credentials/examples/v2"
```

Comment:
```suggestion
```

Reply:
As you can see from the n-quads, this value is not required to generate test vectors for this suite.